### ISSUE TITLE
Reenable kudu-python tests on CI

### ISSUE THREAD

**Issue by cpcloud** [ibis-project/ibis#1530]
After #1525 we'll have reenabled testing for kudu-backed impala tables, but we still don't have tests running with the kudu python client. We should fix that.

**Comment by cpcloud** [comment]
Apparently kudu only does source releases, and it's quite an involved process to build it. Building an entire project like kudu from source just for testing purposes is out of scope IMO.

This leaves an open question about what to do with the existing code that relies on `kudu-python`. I think it's probably safe to remove at this point. I'll post to the pydata mailing list about it and see if anyone chimes in.

**Comment by xmnlab** [comment]
> Building an entire project like kudu from source just for testing purposes is out of scope IMO.

Agreed. If you think kudu is an important backend and if you know anyone how could guide me to create a conda-forge package I can spend sometime on that.

**Comment by cpcloud** [comment]
You could probably synthesize the material here: https://kudu.apache.org/docs/installation.html#build_cpp_client enough to make a conda-forge package. I'm not sure what the maintenance burden for such a package would be.

**Comment by cpcloud** [comment]
@xmnlab Moved this to future, as this shouldn't block the next bugfix release.

**Comment by xmnlab** [comment]
sounds good. thanks @cpcloud !

**Comment by xmnlab** [comment]
PR WIP https://github.com/conda-forge/staged-recipes/pull/8103

**Comment by cpcloud** [comment]
Kudu support is on life-support given the lack of maintainers for it.
